lab/lab1/part3-array-access.html

```diff
@@ -86,9 +86,32 @@
 
     ===================== */
 
-    L.marker([0, 0]).addTo(map);
-    L.marker([0, 0]).addTo(map);
-    L.marker([0, 0]).addTo(map);
+    //Restaurant array
+    restaurantData=[[39.96000445664437, -75.19063211752054, "Sabrina's Cafe"],
+    [39.95228762601507, -75.17471302054324, "JJ Thai Cuisine"],
+    [39.95474788832882, -75.1937667669392, "Sang Kee Noodle House"]];
+
+    //Create markers
+    restaurantData.forEach(function(item) {
+      L.marker([item[0], item[1]]).addTo(map);
+    });
```

Comment:
Very impressed with your consistent (and correct!) usage of `forEach`... Do you have prior experience with higher order functions?